### ISSUE TITLE
unhandled exception on check : coercing to Unicode: need string or buffer, list found

### ISSUE THREAD

**Issue by lucasRolff** [netdata/netdata#7642]
##### Bug report summary

v1.19.0-214-nightly seems to bring an issue with some exception with the LiteSpeed detection, causing the job to be skipped, from the debug:

```
-sh-4.2$ /usr/bin/python /usr/libexec/netdata/plugins.d/python.d.plugin 1 debug litespeed
2019-12-30 20:12:15: python.d INFO: plugin[main] : using python v2
2019-12-30 20:12:15: python.d DEBUG: plugin[main] : looking for 'python.d.conf' in ['/etc/netdata', '/usr/lib/netdata/conf.d']
2019-12-30 20:12:15: python.d DEBUG: plugin[main] : loading '/usr/lib/netdata/conf.d/python.d.conf'
2019-12-30 20:12:15: python.d DEBUG: plugin[main] : '/usr/lib/netdata/conf.d/python.d.conf' is loaded
2019-12-30 20:12:15: python.d DEBUG: plugin[main] : looking for 'pythond-jobs-statuses.json' in /var/lib/netdata
2019-12-30 20:12:15: python.d DEBUG: plugin[main] : loading '/var/lib/netdata/pythond-jobs-statuses.json'
2019-12-30 20:12:15: python.d DEBUG: plugin[main] : '/var/lib/netdata/pythond-jobs-statuses.json' is loaded
2019-12-30 20:12:15: python.d DEBUG: plugin[main] : [litespeed] looking for 'litespeed.conf' in ['/etc/netdata/python.d', '/usr/lib/netdata/conf.d/python.d']
2019-12-30 20:12:15: python.d DEBUG: plugin[main] : [litespeed] loading '/usr/lib/netdata/conf.d/python.d/litespeed.conf'
2019-12-30 20:12:15: python.d DEBUG: plugin[main] : [litespeed] '/usr/lib/netdata/conf.d/python.d/litespeed.conf' is loaded
2019-12-30 20:12:15: python.d INFO: plugin[main] : [litespeed] built 1 job(s) configs
2019-12-30 20:12:15: python.d DEBUG: plugin[main] : litespeed[localhost] was previously active, applying recovering settings
2019-12-30 20:12:15: python.d WARNING: plugin[main] : litespeed[localhost] : unhandled exception on check : TypeError('coercing to Unicode: need string or buffer, list found',), skipping the job
2019-12-30 20:12:15: python.d INFO: plugin[main] : no jobs to serve
2019-12-30 20:12:15: python.d INFO: plugin[main] : exiting from main...
```

Content of the `python.d/litespeed.conf` matches https://github.com/netdata/netdata/blob/master/collectors/python.d.plugin/litespeed/litespeed.conf

##### OS / Environment

CloudLinux release 7.7 (Valery Bykovsky)

##### Netdata version (ouput of `netdata -V`)

netdata v1.19.0-214-nightly

##### Component Name

litespeed python.d.plugin

##### Steps To Reproduce

1: Update to latest nightly
2: Make sure `/tmp/lshttpd/` exists
3: Put an `.rtreport` file in the lshttpd directory with the contents:

```
VERSION: LiteSpeed Web Server/Enterprise/5.4.2
UPTIME: 00:07:10
BPS_IN: 0, BPS_OUT: 0, SSL_BPS_IN: 7, SSL_BPS_OUT: 739
MAXCONN: 10000, MAXSSL_CONN: 10000, PLAINCONN: 0, AVAILCONN: 9950, IDLECONN: 5, SSLCONN: 50, AVAILSSL: 9950
REQ_RATE []: REQ_PROCESSING: 45, REQ_PER_SEC: 46.7, TOT_REQS: 12178, PUB_CACHE_HITS_PER_SEC: 0.7, TOTAL_PUB_CACHE_HITS: 237, PRIVATE_CACHE_HITS_PER_SEC: 0.0, TOTAL_PRIVATE_CACHE_HITS: 2, STATIC_HITS_PER_SEC: 41.2, TOTAL_STATIC_HITS: 10696
BLOCKED_IP:
EOF
```

##### Expected behavior

Job should not be skipped.

**Comment by ilyam8** [comment]
Hi @LucaDev, thx for reporting. Could you test the fix #7645

See 

1. download [`collection.py`](https://raw.githubusercontent.com/ilyam8/netdata/coercing_to_unicode_fix/collectors/python.d.plugin/python_modules/bases/collection.py)
2. move it to the `opt/netdata/usr/libexec/netdata/python.d/python_modules/bases` (path depends on your install)
3. restart `netdata.sevice`

**Comment by lucasRolff** [comment]
@ilyam8 that did the trick:

```
2019-12-31 08:56:02: python.d DEBUG: litespeed[localhost] : stats files: ['/tmp/lshttpd/.rtreport.2', '/tmp/lshttpd/.rtreport']
2019-12-31 08:56:02: python.d INFO: plugin[main] : litespeed[localhost] : check success
CHART netdata.runtime_litespeed_local '' 'Execution time for litespeed_local' 'ms' 'python.d' netdata.pythond_runtime line 145000 10
DIMENSION run_time 'run time' absolute 1 1
```

Thanks a lot!
